src/main/java/org/tndata/android/compass/ui/StreakView.java

```diff
@@ -0,0 +1,79 @@
+package org.tndata.android.compass.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.animation.Animation;
+import android.view.animation.ScaleAnimation;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import org.tndata.android.compass.R;
+import org.tndata.android.compass.model.FeedData;
+
+public class StreakView extends LinearLayout {
+
+    private static final String TAG = "StreakView";
+    private Context mContext;
+    private FeedData.Streak mStreak;
+
+    private LinearLayout mLayoutContainer;
+    private RelativeLayout mStreakBarContainer;
+    private View mStreakBar;
+    private RelativeLayout mImageContainer;
+    private ImageView mStreakCheckImage;
+    private TextView mStreakDay;
+
+    public StreakView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        mContext = context;
+    }
+
+    public void setStreak(FeedData.Streak streak) {
+        mStreak = streak;
+        init();
```

Comment:
Call to init: https://github.com/tndatacommons/android-app/pull/215/files#r71026004